src/conn/conn_layered.c

```diff
@@ -744,9 +744,7 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
         /* Pick up a new checkpoint (followers only). */
         WT_ERR(__wt_config_gets(session, cfg, "disaggregated.checkpoint_meta", &cval));
         if (cval.len > 0) {
-            if (leader)
```

Comment:
When we step down, step up, and step down again, we will hit this code as we have "disaggregated.checkpoint_meta" in the base configuration. I don't know what is the best way to fix this. @pmacko86

Reply:
added a comment